python/ray/_raylet.pyx

```diff
@@ -1619,8 +1625,11 @@ cdef class CoreWorker:
                 c_actor_id,
                 ray_function,
                 args_vector, CTaskOptions(name, num_returns, c_resources))
-
-            return VectorToObjectRefs(return_refs)
+            if return_refs.has_value():
+                return VectorToObjectRefs(return_refs.value())
+            else:
+                raise BackPressureError("Back pressure occurs "
```

Comment:
This error message is not great. We should be more precise about the error. Please refer to https://spark.apache.org/error-message-guidelines.html. Maybe alternatively, we can say something like 

```
Cannot submit the actor task {method_name} of actor {repr(actor)} because there are X number of pending tasks queued that exceeds the limit Y. To increase the max number of queued tasks to the actor, do Z. 
```

Reply:
cpp unit test added. error message improved.